lib/std/fs/test.zig

```diff
@@ -1404,3 +1404,69 @@ test "File.PermissionsUnix" {
     try testing.expect(permissions_unix.unixHas(.user, .execute));
     try testing.expect(!permissions_unix.unixHas(.other, .execute));
 }
+
+test "preopens" {
+    if (builtin.os.tag != .wasi) return error.SkipZigTest;
+
+    // lifted from `testing`
+    const random_bytes_count = comptime (12);
+    const sub_path_len = comptime (std.fs.base64_encoder.calcSize(random_bytes_count));
+    const path = comptime ("/tmp");
+    const tmp_file_name = comptime ("file.txt");
+    const nonsense = comptime ("nonsense");
```

Comment:
Those usages of `comptime` are reduntant except for the call to `calcSize` as function calls are always runtime unless in a comptime scope.

Reply:
ah, `calcSize` must be what threw me off and made me defensive